swc/private/swc.bzl

```diff
@@ -23,10 +23,18 @@ _attrs = {
         """,
     ),
     "source_maps": attr.string(
-        doc = "see https://swc.rs/docs/usage/cli#--source-maps--s",
+        doc = """Create source map files for emitted JavaScript files.
+
+        see https://swc.rs/docs/usage/cli#--source-maps--s""",
         values = ["true", "false", "inline", "both"],
         default = "false",
     ),
+    "source_root": attr.string(
+        doc = """Specify the root path for debuggers to find the reference source code.
+
+        see https://swc.rs/docs/usage/cli#--source-root""",
+        default = "",
```

Comment:
this feels wrong, why have a default value here? that makes it seem like we would pass an empty string to this flag by default. Instead it should just be optional